Wikimate.php

```diff
@@ -12,6 +12,10 @@
 /**
  * Provides an interface over wiki API objects such as pages and files.
  *
+ * All requests to the API can throw an exception if the server is lagged
+ * and a finite number of retries is exhausted.  By default requests are
+ * tried indefinitely.  See {@see Wikimate::request()} for more information.
```

Comment:
Just a nitpick, may make this a little clearer:

```suggestion
 * retried indefinitely.  See {@see Wikimate::request()} for more information.
```